module/apmfasthttp/server_test.go

```diff
@@ -64,8 +65,24 @@ func TestServerHTTPResponse(t *testing.T) {
 	resp.Body.Close()
 	assert.Equal(t, fasthttp.StatusUnauthorized, resp.StatusCode)
 	tracer.Flush(nil)
-
 	payloads := transport.Payloads()
+
+	timer := time.NewTimer(100 * time.Millisecond)
```

Comment:
Can you please add a comment above this to explain why it's needed, and that it's particular to fasthttp? I wouldn't want anyone to cargo cult it